### PR TITLE
obis-observer: add commands

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/obis-observer/commands/uplink/getSerialPort.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/**
+ * Uplink command to get the information about serial port settings.
+ *
```
Comment (DarkPark) [inline src/obis-observer/commands/uplink/getSerialPort.ts:3]: @packageDocumentation

File: src/obis-observer/commands/uplink/getSettingsMemory.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/**
+ * Uplink command to get the information about memory settings.
+ *
```
Comment (DarkPark) [inline src/obis-observer/commands/uplink/getSettingsMemory.ts:3]: @packageDocumentation

File: src/obis-observer/commands/uplink/observationReport.ts [hunk 3]
```diff
@@ -0,0 +1,120 @@
+/**
+ * The Observation Report uplink command is used for reporting by the OBIS observer.
+ * The command includes the OBIS content captured according to schedule and contain float values.
+ *
```
Comment (DarkPark) [inline src/obis-observer/commands/uplink/observationReport.ts:4]: @packageDocumentation

File: src/obis-observer/commands/uplink/observationReportString.ts [hunk 4]
```diff
@@ -0,0 +1,129 @@
+/**
+ * The Observation Report String uplink command is used for reporting by the OBIS observer.
+ * The command includes the OBIS content captured according to schedule and contain string values.
+ *
```
Comment (DarkPark) [inline src/obis-observer/commands/uplink/observationReportString.ts:4]: @packageDocumentation

**Review by aleks-and — APPROVED** [review]

File: src/obis-observer/commands/uplink/getSerialPort.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/uplink/getSerialPort.ts:3]: fixed

File: src/obis-observer/commands/uplink/getSettingsMemory.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/uplink/getSettingsMemory.ts:3]: fixed

File: src/obis-observer/commands/uplink/observationReport.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/uplink/observationReport.ts:4]: fixed

File: src/obis-observer/commands/uplink/observationReportString.ts (hunk 4 above)
Comment (DeadByElpy) [reply src/obis-observer/commands/uplink/observationReportString.ts:4]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**